integration/testcmd/dhclient/main.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013-2019 the Webboot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"github.com/u-root/webboot/pkg/dhclient"
+)
+
+func main() {
+	dhclient.New("^e.*", 15, 5, false, true, true)
```

Comment:
I don't think you need to make the others flags, but I think it would be nice to make ipv4 and ipv6 flags, then default them to true.

Reply:
This way you can add tests for different configurations to test in the future.

Reply:
added some flags to this file. I made flags such as ipv4, ipv6, retry, and timeout to give people the options to try a few different things in the future. However, i left things like verbose set to true always and not as a flag as well as the interface name to "^e.*"